checkov/main.py

```diff
@@ -22,27 +22,49 @@
 from checkov.serverless.runner import Runner as sls_runner
 from checkov.terraform.plan_runner import Runner as tf_plan_runner
 from checkov.terraform.runner import Runner as tf_runner
+from checkov.helm.runner import Runner as helm_runner
 from checkov.version import version
 
 outer_registry = None
 
 logging_init()
+checkov_runner_module_names = ['cfn', 'tf', 'k8', 'sls', 'arm', 'tf_plan', 'helm']
+checkov_runners = ['cloudformation', 'terraform', 'kubernetes', 'serverless', 'arm', 'terraform_plan', 'helm']
 
+#Check runners for system deps, append --skip-framework list for failing deps.
+checkov_frameworks_unmatched_deps = []
+for runner in checkov_runner_module_names:
```

Comment:
maybe add those to a python file - something like "runner_dependency_handler" with a short pydoc explaining how to use/ add

Reply:
Todo

Reply:
Moved this to a little class as well as the logic that appends the args.skipped_framework, keeps `main.py` cleaner as discussed. also added pydoc.